apps/condo/domains/billing/schema/BillingCategory.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Generated by `createschema billing.BillingCategory 'name:Text'`
+ */
+
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/billing/access/BillingCategory')
+const { LocalizedText } = require('@core/keystone/fields')
+
+
+const BillingCategory = new GQLListSchema('BillingCategory', {
+    schemaDoc: 'Payment category - used primarily in display purposes',
```

Comment:
Is it a part of integration responsibility? It's also used to allow to have two different receipts for one billing account

Reply:
1. Yes, integrations should choose one of the available billing categories for each receipt, they create or update.
2. As of now, you can have a lot of receipts linked to one billing account. But they usually have different recipients.